src/WingetCreateCLI/Program.cs

```diff
@@ -71,7 +71,11 @@ private static async Task<int> Main(string[] args)
                 }
                 else
                 {
-                    return 1;
+                    // Do not block creating a new manifest if loading the GitHub client fails. InstallerURL could point to a local network.

```

Comment:
Is local network case common? Will this expose more issues than it solves for some uncommon cases?